### ISSUE TITLE
Reproduction model perform poorly on real data

### ISSUE THREAD

**Issue by q935970314** [IceClear/StableSR#64]
Thanks for sharing such great work！

On the synthetic data, the processing results of the model I reproduced and the pretrain model you provided are basically the same. 
pretrain-117   |   reproduced-125
syn-input       |   ground-truth
![2023-08-01 13-53-13 的屏幕截图](https://github.com/IceClear/StableSR/assets/35994018/c1df4bf1-0bda-4705-8382-cce3ae687f62)

However, the difference in real data is huge, the effect of the pretrain model is still very good, and the reproduced model is basically ineffective (I keep the processing scripts consistent and only change the weight of the model). 
I tried to downsample the real data by 4 times, and then upsample back to the original size, and the reproduction model worked. 
pretrain-117              |    reproduced-125-downup
reproduced-125        |    real-input
![2023-08-01 13-28-32 的屏幕截图](https://github.com/IceClear/StableSR/assets/35994018/9b7489e7-4b6c-4ff3-a6e8-98856b4c3e4c)

I feel this is caused by insufficient generalization of synthetic data during training or other reasons. 
When training, I followed your code exactly and made no modifications. Could you please give me some advice?

**Comment by 543877815** [comment]
Could you point out which part is not as good as you expected?

**Comment by q935970314** [comment]
> Could you point out which part is not as good as you expected?

Sorry. I put the subtitle on the figure so that the contrast is clearer. 
The main problem is that the reconstruction model does not work well on the real test data, as shown in the middle and right sides of the figure below. The pre-trained model can handle it very well, as shown on the left side of the figure below. But they all do well on synthetic data.
![wkppoKbNwN](https://github.com/IceClear/StableSR/assets/35994018/46138f8c-2b3c-4cc2-bc68-32f0ad14a071)

**Comment by stalin18** [comment]
@q935970314 Could you provide more information about how were you able to replicate the results on the synthetic dataset at epoch 125? Did you use the same configs and the dataset for training?

**Comment by q935970314** [comment]
> @q935970314 Could you provide more information about how were you able to replicate the results on the synthetic dataset at epoch 125? Did you use the same configs and the dataset for training?

Yes, I use exactly the settings in the code. Same config, same dataset, same GPU

**Comment by q935970314** [comment]
@IceClear Hi, do you have any suggestions?
Did you use other data synthesis methods during training?

**Comment by IceClear** [comment]
Sry for the late reply. I am now busy with other projects.
I am not sure of the reasons. I did not use additional data.
We noticed that sometimes the training ckpt may be confined to the training data and carefully choosing the ckpt may be required.

**Comment by xiezheng-cs** [comment]
> Sry for the late reply. I am now busy with other projects. I am not sure of the reasons. I did not use additional data. We noticed that sometimes the training ckpt may be confined to the training data and carefully choosing the ckpt may be required.

I have carefully chosen the ckpt and tested all checkpoints, but it is still worse than the public stablesr_000117.ckpt.
